### DISCUSSION TITLE
Проблемы с archlinux.org (возможно не только он)

### DISCUSSION THREAD

**Discussion by Guru1-228** [bol-van/zapret#783] [Q&A]
Началось с того, что не смог скачать пакеты. Потом понял, что сайт не работает, нашёл несколько сообщений о таких же проблемах. С впн работает. Блокчек по нему не даёт результата
Сюда решил заспамить, чтобы узнать о массовости проблемы и поиске решений

На всякий случай уточняю, что с пакманом всё норм, а вот аур отвалился, как и сам сайт арчлинукса

**Comment by you-oops-dev** [comment]
Пишите какой провайдер у вас и провод или мобильный указать при одноименности провайдера.
> Началось с того, что не смог скачать пакеты

Пакеты с archlinux.org не скачиваются исключения только AUR пакеты поскольку AUR Helper стучится на aur.archlinux.org. Как минимум делает реквест и скачивает "исходники" посредством git clone.

Конкретно с пакетами, которые в репо надо выяснить на какое зеркало pacman случался. Стучится он на зеркала, которые указаны в `/etc/pacman.d/mirrorlist`. Вроде бы порядок определяется позицией т.е если первое указано первым он к нему и стучится. Если pacman не скачивает ничего или скачивание медленное переходит ко второму зеркалу.

**Reply by Guru1-228** [reply]
Чита, ростелеком (провод) и мегафон (мобильный) не заходят на сайт

> Пакеты с archlinux.org не скачиваются исключения только AUR пакеты поскольку AUR Helper стучится на aur.archlinux.org

Потому я и откопал, что проблемы с доступом на archlinux.org

**Reply by you-oops-dev** [reply]
> Чита, ростелеком (провод)
Интересно, кажись и на провод перекинулась "зараза".
Так, что в момент скачивание было какой провайдер мегафон или Ростелеком?

**Reply by Guru1-228** [reply]
Ну комп же по кабелю подключен, так что ростелеком, уже потом по мобильному проверил
Ещё "сходил" на работу, там ростелеком, и там работает

**Reply by you-oops-dev** [reply]
> Ну комп же по кабелю подключен

Комп может и мобильный использовать.

> Ещё "сходил" на работу, там ростелеком, и там работает

Ну значит вы такой везучий. Больше новостей читать надо.  И там скорее всего ответ был дан РКН с Hetzer (сайт Арча там) играется не понятно на совсем или только так попугать.
Проблеме как минимум примерно 7 дней.
Как я понял все это региональное и как я понял больше всех коснулось мобильного интернета. У меня пока все тихо с мобильного (сегодня еще не проверял) и проводного.

Че ж можно сказать запасайтесь проксями и VPN.

**Reply by Guru1-228** [reply]
Я так понял тут по ip блокировка, а не dpi поработал?

**Reply by you-oops-dev** [reply]
> Я так понял тут по ip блокировка ...

Скорее всего даже по ASN. Может я даже и ошибаюсь. Может что-то DPI тригерит например доменная зона (org,net и т.д) или версия TLS. В общем 7 дней прошло, а так и никто и не понял толком. Официального ответа нет и не будет скорее всего.

> а не dpi поработал

Ну так и не будет в листе его нет потому, что его в реестре нет.

**Reply by Guru1-228** [reply]
Лист я сам составляю, однако с этим сайтом не срабатывает

**Reply by you-oops-dev** [reply]
> Лист я сам составляю, однако с этим сайтом не срабатывает

Удачи можно только пожелать с учетом того, что Hetzer это 2720064 адреса IPv4, а получить все поддомены и домены невозможно.

**Reply by Guru1-228** [reply]
Ну в лист я не пытаюсь внести все блокировки мира, а только то чем пользуюсь или потенциально могу воспользоваться, а следовательно список не столь огромный
